plugins/a11ychecker.md

```diff
@@ -207,6 +207,6 @@ var issues = editor.plugins.a11ychecker.getReport();
 
 ## Downloading Accessibility Checker
 
-A [TinyMCE Enterprise](https://www.tinymce.com/pricing/) subscription includes the ability to download and install the accessibility checker feature for the editor.
+A [{{site.cloudname}}]({{site.pricingpage}}) subscription includes the ability to download and install the accessibility checker feature for the editor.
```

Comment:
Should this be `site.sfenterpriseac` since cloud can't be downloaded?

Reply:
And likewise for the other `plugins` pages?

Reply:
fixed